src/applications/disability-benefits/view-payments/components/view-payments-lists/payments-received/PaymentsReceived.jsx

```diff
@@ -1,46 +1,47 @@
-import React from 'react';
+import React, { Component } from 'react';
```

Comment:
I think this Component import can be removed.

Reply:
removed